src/xGadgetron/cGadgetron/gadgetron_data_containers.cpp

```diff
@@ -549,17 +550,17 @@ AcquisitionsFile::take_over(MRAcquisitionData& ac)
 {
 	AcquisitionsFile& af = (AcquisitionsFile&)ac;
 	acqs_info_ = ac.acquisitions_info();
-	if (index_)
-		delete[] index_;
+	if (index_.size() > 0)
+		index_.resize(0);
```

Comment:
we don't need these 2 lines anymore as far as I can see. The code below will always re-assign or resize. We don't need to first resize. 2 lines less :-)

Reply:
Done, also did this in some other places where there was unnecessary resizing.